runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -123,11 +123,14 @@ static void wrtbarEvaluator(TR::Node *node, TR::Register *srcReg, TR::Register *
       {
       generateCompareBranchInstruction(cg, TR::InstOpCode::cbzx, node, srcReg, doneLabel, NULL);
       }
-   generateImmSymInstruction(cg, TR::InstOpCode::bl, node, (uintptr_t)wbRef->getMethodAddress(),
+   TR::Instruction *gcPoint = generateImmSymInstruction(cg, TR::InstOpCode::bl, node, (uintptr_t)wbRef->getMethodAddress(),
                                         new (cg->trHeapMemory()) TR::RegisterDependencyConditions((uint8_t)0, 0, cg->trMemory()),
                                         wbRef, NULL);
+   gcPoint->ARM64NeedsGCMap(cg, 0xFFFFFFFF);
```

Comment:
None of the non-realtime write barriers are GC points, so a GC map is not needed here (and realtime GC is not supported on AArch64).

Reply:
Thank you for your explanation.
Removed the call to `ARM64NeedsGCMap`.